packages/fether-react/package.json

```diff
@@ -31,9 +31,10 @@
     "start": "npm-run-all -p start-*",
     "start-css": "npm run build-css -- --watch --recursive",
     "start-js": "cross-env SKIP_PREFLIGHT_CHECK=true BROWSER=none craco start --react-scripts ../../node_modules/react-scripts",
-    "test": "ln -s ../../../node_modules/react-scripts node_modules/ && cross-env SKIP_PREFLIGHT_CHECK=true craco test; rm node_modules/react-scripts"
+    "test": "jest --all --color --coverage"
```

Comment:
Keep craco test, because we remove all babel plugins.

Reply:
I'm going to need help with this...

It seems that the symlink to `../../../node_modules/react-scripts` in the existing package.json of fether-react actually corresponds to one directory higher than the root directory. And there wasn't anything to run any fether-electron tests previously since there weren't any, but I've added some now. If I change it to `../../node_modules/react-scripts` in both fether-electron and fether-react's package.json files and borrow the code that you did in this PR https://github.com/paritytech/fether/pull/404 so it becomes `"test": "ln -sf ../../node_modules/react-scripts node_modules/ && cross-env SKIP_PREFLIGHT_CHECK=true craco test"`, then it gives me the following error when I run `yarn && yarn test`:

<details><p><pre>
scon @ ~/code/src/paritytech/fether - [luke-312-taskbar-app] $ yarn && yarn test
yarn install v1.10.1
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 1.99s.
yarn run v1.10.1
$ semistandard '**/*.{js,ts}' --parser babel-eslint && CI=true lerna run test --parallel
lerna notice cli v3.10.7
lerna info ci enabled
lerna info Executing command in 2 packages: "yarn run test"
fether: $ ln -sf ../../node_modules/react-scripts node_modules/ && cross-env SKIP_PREFLIGHT_CHECK=true craco test
fether-react: $ ln -sf ../../node_modules/react-scripts node_modules/ && cross-env SKIP_PREFLIGHT_CHECK=true craco test
fether-react: internal/modules/cjs/loader.js:611
fether-react:     throw err;
fether-react:     ^
fether-react: Error: Cannot find module '/Users/scon/code/src/paritytech/fether/packages/fether-react/node_modules/react-scripts/config/env.js'
fether-react:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:609:15)
fether-react:     at Function.resolve (internal/modules/cjs/helpers.js:28:19)
fether-react:     at resolveConfigFilePath (/Users/scon/code/src/paritytech/fether/node_modules/@craco/craco/lib/cra.js:7:20)
fether-react:     at Object.<anonymous> (/Users/scon/code/src/paritytech/fether/node_modules/@craco/craco/lib/cra.js:21:9)
fether-react:     at Module._compile (internal/modules/cjs/loader.js:734:30)
fether-react:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
fether-react:     at Module.load (internal/modules/cjs/loader.js:626:32)
fether-react:     at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
fether-react:     at Function.Module._load (internal/modules/cjs/loader.js:558:3)
fether-react:     at Module.require (internal/modules/cjs/loader.js:663:17)
fether: internal/modules/cjs/loader.js:611
fether:     throw err;
fether:     ^
fether: Error: Cannot find module '/Users/scon/code/src/paritytech/fether/packages/fether-electron/node_modules/react-scripts/config/env.js'
fether:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:609:15)
fether:     at Function.resolve (internal/modules/cjs/helpers.js:28:19)
fether:     at resolveConfigFilePath (/Users/scon/code/src/paritytech/fether/node_modules/@craco/craco/lib/cra.js:7:20)
fether:     at Object.<anonymous> (/Users/scon/code/src/paritytech/fether/node_modules/@craco/craco/lib/cra.js:21:9)
fether:     at Module._compile (internal/modules/cjs/loader.js:734:30)
fether:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
fether:     at Module.load (internal/modules/cjs/loader.js:626:32)
fether:     at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
fether:     at Function.Module._load (internal/modules/cjs/loader.js:558:3)
fether:     at Module.require (internal/modules/cjs/loader.js:663:17)
fether: error Command failed with exit code 1.
fether-react: error Command failed with exit code 1.
fether: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
fether-react: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run test exited 1 in 'fether'
lerna ERR! yarn run test stdout:
$ ln -sf ../../node_modules/react-scripts node_modules/ && cross-env SKIP_PREFLIGHT_CHECK=true craco test
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run test stderr:
internal/modules/cjs/loader.js:611
    throw err;
    ^

Error: Cannot find module '/Users/scon/code/src/paritytech/fether/packages/fether-electron/node_modules/react-scripts/config/env.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:609:15)
    at Function.resolve (internal/modules/cjs/helpers.js:28:19)
    at resolveConfigFilePath (/Users/scon/code/src/paritytech/fether/node_modules/@craco/craco/lib/cra.js:7:20)
    at Object.<anonymous> (/Users/scon/code/src/paritytech/fether/node_modules/@craco/craco/lib/cra.js:21:9)
    at Module._compile (internal/modules/cjs/loader.js:734:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
    at Module.load (internal/modules/cjs/loader.js:626:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
    at Function.Module._load (internal/modules/cjs/loader.js:558:3)
    at Module.require (internal/modules/cjs/loader.js:663:17)
error Command failed with exit code 1.

lerna ERR! yarn run test exited 1 in 'fether'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
</pre><p></details>

I get this same error both when I keep my changes to packages/fether-electron/babel.config.js and /packages/fether-react/babel.config.js, and also when I delete those files.

I get the same error if I remove all Babel and Jest plugins from fether-electron and fether-react (except for leaving @babel/plugin-proposal-decorators in the package.json of fether-react as you mentioned)

Reply:
I've got it working now